extensions/AntiSpoof/SpoofUser.php

```diff
@@ -45,22 +45,32 @@ public function getNormalized() {
 	/**
 	 * Does the username pass Unicode legality and script-mixing checks?
 	 *
+	 * @param $skipExactMatch - when true exact match username is not returned
+	 *
 	 * @return array empty if no conflict, or array containing conflicting user names
 	 */
-	public function getConflicts() {
+	public function getConflicts( $skipExactMatch = false ) {
 		$dbr = self::getDBSlave();
 
 		// Join against the user table to ensure that we skip stray
 		// entries left after an account is renamed or otherwise munged.
-		/* Wikia Change - begin : Quote the table names, otherwise the select method
-		   tries to prefix the tablenames with the current user DB */
+		/* Wikia Change - begin :
+			Quote the table names, otherwise the select method
+				tries to prefix the tablenames with the current user DB
+			Added $skipExactMatch */
+		$conds = array(
+			'su_normalized' => $this->mNormalized,
+			'su_name=user_name',
+		);
+		if ( $skipExactMatch ) {
```

Comment:
This method is used during user rename process, so we need to change it conditionally.